Drivers/Src/MotorController.c

```diff
@@ -11,13 +12,71 @@
 #define MASK_CC_ERR 1<<2 //checks velocity on 2 bit
 #define MASK_OVER_SPEED_ERR 1<<24 //check if motor overshot max RPM on 24 bit
```

Comment:
I think we should put these inside of parentheses or declare them as `const uint32_t`, so we don't get any nasty surprises when the preprocessor pastes these in and the `<<` operator has lower precedence than some other operator